packages/functional-tests/tests/misc.spec.ts

```diff
@@ -28,3 +28,13 @@ test.describe('severity-1', () => {
     expect(await relier.isLoggedIn()).toBe(true);
   });
 });
+
+test.describe('robots.txt', () => {
```

Comment:
I'd written this one a while ago, I can remove if we no longer want this test but we don't have any unit tests for it so it seems good to leave to me?

Reply:
We can leave it on here!